VMBackup/main/common.py

```diff
@@ -42,6 +42,9 @@ class CommonVariables:
     statusBlobUploadError = 'statusBlobUploadError'
     TempStatusFileName = 'tempStatusFile.status'
     onlyLocalFilesystems = 'onlyLocalFilesystems'
+    isSnapshotTTLEnabled = 'isSnapshotTTLEnabled'
+    useMCCFForLAD = 'UseMCCFForLAD'
+    useMCCFToFetchDSASForAllDisks = 'UseMCCFToFetchDSASForAllDisks'
```

Comment:
Same. Lets follow https://microsoft.sharepoint.com/:p:/t/ComputeVM/ETy6F_4NonBIlidU43WLAHMBpwdAo3k3sTatzaHIhdWOUw?e=8QCmVe&nav=eyJzSWQiOjI1OSwiY0lkIjoyMzY4Mjk5MzM5fQ just for uniformity's sake.